logstash-core/src/test/resources/org/logstash/plugins/AliasRegistry.yml

```diff
@@ -0,0 +1,9 @@
+input:
+  aliased_input1: beats
+  aliased_input2: tcp
+
+filter:
+  aliased_filter: json
+
+
+
```

Comment:
nit: a single trailing newline is sufficient.

Reply:
done